ldpd/lde.h

```diff
@@ -147,6 +147,7 @@ void		 lde_imsg_compose_parent_sync(int, pid_t, void *, uint16_t);
 int		 lde_imsg_compose_ldpe(int, uint32_t, pid_t, void *, uint16_t);
 int		 lde_acl_check(char *, int, union ldpd_addr *, uint8_t);
 uint32_t	 lde_update_label(struct fec_node *);
+void		 lde_free_label(uint32_t);
```

Comment:
Could this be `uint32_t label` ? (I realize there are other examples here - but the preferred style now is to include an argument name for each arg)

Reply:
+1, please include identifier names.